en/config/actuators.md

```diff
@@ -313,6 +313,51 @@ Configuring the Camera Capture / Trigger input requires a reboot to take effect
 You should assign functions to the outputs that match your physical wiring of motors and servos, and use the [Actuator Testing](#actuator-testing) section described below to determine appropriate output parameter values.
 These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
+### Output Functions
+
+Output functions are used to map the "logical functions" of an airframe, such as `Motor 1` or `Landing gear`, to physical outputs like FMU output pin 2.
+This makes it easy to use a particular output pin for almost any purpose.
+
+Some functions are only relevant to particular frames or output types, and will not be offered on others.
```

Comment:
This is specified in the Mixer of the control_allocator's [parameter definition yaml file](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/control_allocator/module.yaml#L452-L868). Maybe worth mentioning? Or is it too much information for the users?

Reply:
I mentioned the specific cases that are constrained alongside the outputs, and link to the file at the end. Generally I don't think people need to know too much here other than that they might not see an output all the time.